From 4fb6bbfbc53428fe5d38ec3e04bb4fd187b2fd45 Mon Sep 17 00:00:00 2001
From: Derek Straka <derek@asterius.io>
Date: Fri, 22 Dec 2017 09:07:00 -0500
Subject: [PATCH] setup.py: remove the dependency on m2r

The dependency on m2r is removed here as it only provides the
long_description value that is optional.  This item is just the
text that would be present on PyPi, so it doesn't provide much
value in this case

Upstream-Status: Inappropriate (OE specific)

Signed-off-by: Derek Straka <derek@asterius.io>
Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 setup.py | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/setup.py b/setup.py
index 3fb7fdb..245d69a 100644
--- a/setup.py
+++ b/setup.py
@@ -4,14 +4,6 @@ Setup file for automat
 
 from setuptools import setup, find_packages
 
-try:
-    from m2r import parse_from_file
-    long_description = parse_from_file('README.md')
-except(IOError, ImportError):
-    print("\n\n!!! m2r not found, long_description is bad, don't upload this to PyPI !!!\n\n")
-    import io
-    long_description = io.open('README.md', encoding="utf-8").read()
-
 setup(
     name='Automat',
     use_scm_version=True,
@@ -19,12 +11,10 @@ setup(
     description="""
     Self-service finite-state machines for the programmer on the go.
     """.strip(),
-    long_description=long_description,
     packages=find_packages(exclude=[]),
     package_dir={'automat': 'automat'},
     setup_requires=[
         'setuptools-scm',
-        'm2r',
     ],
     install_requires=[
         "attrs>=19.2.0",
-- 
2.17.1

